os2mo_data_import/os2mo_helpers/mora_helpers.py

```diff
@@ -532,14 +539,19 @@ def read_organisation_people(self, org_uuid, person_type='engagement',
                 all_persons = all_persons + persons
 
         for person in all_persons:
+            if len(primary_types) != 0 and person['primary']['uuid'] not in primary_types:
```

Comment:
Den boolsk evaluering på en tom liste er falsk, en liste med indhold evaluere sand:
```
>>> bool([])
False
>>> bool([0])
True
```
Man kan altså droppe det komplekse tjek her, til noget simplere:
```suggestion
            if primary_types and person['primary']['uuid'] not in primary_types:
```